styles.css

```diff
@@ -1259,4 +1420,49 @@ textarea:focus {
     margin-left: 32px;
     opacity: 0;
   }
+
+  .image-container {
+    width: 311px;
+    height: 220px;
+    top: 156px;
+  }
+
+  .btn-x {
+    cursor: pointer;
+    border: none;
+    outline: none;
+    font-size: 1.5rem;
+    margin-left: 300px;
+    margin-top: -28px;
+    margin-right: 30px;
+  }
+}
+
+@media (max-width: 368px) {
+  .btn-x {
+    margin-left: 200px;
+  }
 }
+
+@media (min-width: 1228px) {
+  /* .description {
+    width: 524px;
+    height: 336px;
+    margin-right: 27px;
+  } */
+
+  /* .modal-button {
+    margin-top: -36px;
+    margin-left: 732px;
+  } */
+}
+
+/* @media only screen and (max-width: 1226px) and (min-width: 1106px) {
+  .description {
+    margin-right: 40px;
+  }
+
+  .image-container {
+    width: 541px;
+  }
+} */
```

Comment:
- [ ] [OPTIONAL]  I would like to recommend you remove all the commented code from your codebase in order to keep your code clean :smile: